Gemfile.lock

```diff
@@ -2,21 +2,21 @@ PATH
   remote: .
   specs:
     danger-dangermattic (0.0.1)
-      danger (~> 9.3)
+      danger (~> 9.4)
```

Comment:
Wow I'm surprised danger Ruby got an update, I thought it wasn't that much active nowadays 😅

Reply:
[It did](https://github.com/danger/danger/releases/tag/v9.4.3)!
This latest version contains [a fix I implemented](https://github.com/danger/danger/pull/1477) for a bug we've been experiencing: in-line comments from Danger, such as the RuboCop or SwiftLint ones, not being removed.